packages/animation/src/parts/simple/zoom.ts

```diff
@@ -17,30 +17,32 @@
 /**
  * Internal dependencies
  */
-import { ANIMATION_TYPES } from '../../constants';
-import SimpleAnimation from '../simpleAnimation';
+import { AMPEffectTiming, AnimationType, FieldType } from '../../types';
+import createAnimation from '../createAnimation';
 
-const defaults = {
-  fill: 'forwards',
-  duration: 1000,
-};
+const defaults: AMPEffectTiming = { fill: 'forwards', duration: 1000 };
```

Comment:
Nit: `forwards` looks like something that would be nice to move to an enum. Looks like there are only a few available values for `fill` in the code.

Reply:
We only have that as a [string union type from the web animations package](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/web-animations-js/index.d.ts#L6), but it will be validated against this union type so we should be safe.